### ISSUE TITLE
run_immediately doesn't :)

### ISSUE THREAD

**Issue by P-T-I** [zmap/celerybeat-mongo#58]
Setting the boolean field run_immediately from false to true does not start the task at all. The only thing I see is that after I set the field to 'true' that several seconds later the run_immediately field is set back to 'false'. Logs show no task activity, the task involved is not started directly; it only follows the configured crontab schedule.

**Comment by rafaelreuber** [comment]
Only celerybeat-mongo have such feature. I have checked the others ones (django-celery-beat and redbeat). I think this not so much useful. It's so easy to run task immediately.

**Comment by P-T-I** [comment]
I agree it is trivial however in the is_due method there are checks made for that specific setting so I reconned there should be a supported functionality behind, and if that’s the case; it’s not working...

**Comment by rafaelreuber** [comment]
You are right, if the feature is available, it must work perfectly!

**Comment by P-T-I** [comment]
Can you have a look at it, or shall I?

**Comment by P-T-I** [comment]
@rafaelreuber, I’ve taken a look at this particular issue, looks like the functionality only works when (re)starting the daemon. Then and only then it sees documents with the “run_immediately” set to true, all “is_due” cycles after that it never rechecks the “run_immediately” set in the database...

**Comment by FlorianPfisterer** [comment]
@rafaelreuber Can you provide some hints on how to solve this? I'm happy to make a PR, but I'm not sure where to start.
